### PR TITLE
feat: CrowdAction Subscribe & Unsubscribe

### PR DESCRIPTION
**Pull request by wizlif**

### CODE REVIEW

File: lib/application/crowdaction/subscription/status/subscription_status_bloc.dart [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import 'dart:async';
```
Comment (Xazin) [inline lib/application/crowdaction/subscription/status/subscription_status_bloc.dart:1]: Should we move some files around a bit?
`application/crowdaction/subscription_status/`
`application/crowdaction/subscription/`
Just a suggestion!

Comment (wizlif) [reply lib/application/crowdaction/subscription/status/subscription_status_bloc.dart:1]: Like keep the blocs in the same folder ??

Comment (Xazin) [reply lib/application/crowdaction/subscription/status/subscription_status_bloc.dart:1]: No, but right now you have it like this:

lib/application/crowdaction/subscription/
  - bloc_files
  - /status/
    - bloc_files

And it should be:
lib/application/crowdaction/subscription/
  - bloc_files
lib/application/crowdaction/subscription_status/
  - bloc_files

Comment (Xazin) [reply lib/application/crowdaction/subscription/status/subscription_status_bloc.dart:1]: You have a folder with a bloc and inside that bloc's folder you have a folder to another bloc, and they aren't related to each other logic wise, there's no dependency.

**Review by Xazin — APPROVED** [review]

**Pull request merged**